compiler-rt/lib/sanitizer_common/sanitizer_linux.cpp

```diff
@@ -1180,7 +1180,7 @@ uptr ReadBinaryName(/*out*/ char *buf, uptr buf_len) {
   uptr module_name_len = internal_readlink(default_module_name, buf, buf_len);
   int readlink_error;
   bool IsErr = internal_iserror(module_name_len, &readlink_error);
-#    endif  // SANITIZER_SOLARIS
+#    endif  // SANITIZER_FREEBSD || SANITIZER_NETBSD
```

Comment:
I'd rather drop then and use indentation to find real one

Reply:
ok